uniffi_bindgen/src/main.rs

```diff
@@ -66,17 +78,25 @@ fn main() -> Result<()> {
             .arg(clap::Arg::with_name("cdylib_dir").required(true).help("Path to the directory containing the cdylib the scripts will be testing against."))
             .arg(clap::Arg::with_name("idl_file").required(true))
             .arg(clap::Arg::with_name("test_scripts").required(true).multiple(true).help("Foreign language(s) test scripts to run"))
+            .arg(
+                clap::Arg::with_name("config")
+                .long("--config-path")
+                .takes_value(true)
+                .help("Path to the optional uniffi config file. If not provided, uniffi-bindgen will try to guess from the IDL's file location.")
+            )
```

Comment:
This arg is repeated for each subcommand. Nit: make the arg an arg of the `App`.